Netable/Example/ViewModels/HomeVM.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  HomeVM.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-23.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+import Netable
+
+class HomeVM: ObservableVM {
+    @Published var title: String = ""
+    @Published var content: String = ""
+    @Published var posts: [Post] = []
+
+    override func bindViewModel() {
+        super.bindViewModel()
+        getPosts()
+    }
+
+    func getPosts() {
+        Task { @MainActor in
+            posts = try await AuthNetworkService.shared.getPosts()
+        }
+    }
+
+    func createPost() {
+        Task { @MainActor in
```

Comment:
Nitpicky, but does this need to be run on the main thread? AFAICT it doesn't update any UI?

Reply:
It doesn't and I'll go ahead and remove it. I was initially planning on having it update until I realized that it just wasn't practical given our mock api situation.

Reply:
It also shouldn't be a try await because it's not an async or throwing function, so I fixed that as well.